src/Files.App/UserControls/KeyboardShortcut/KeyboardShortcut.cs

```diff
@@ -28,10 +22,24 @@ private void OnSizeChanged()
 		{
 		}
 
-		private void OnHotKeysChanged()
+		private async void OnHotKeysChanged()
 		{
+			var keyboardShortcutItemsControl = GetTemplateChild(KeyboardShortcutItemsControl) as ItemsControl;
+			for (int i = 0; i < 100 && keyboardShortcutItemsControl is null; i++)
+			{
+				// Wait for KeyboardShortcutItemsControl to be loaded
+				await Task.Delay(10);
+				keyboardShortcutItemsControl = GetTemplateChild(KeyboardShortcutItemsControl) as ItemsControl;
+			}
+
+			if (keyboardShortcutItemsControl is null)
+				return;
```

Comment:
It seems that it may take a little while before the KeyboardShortcutItemsControl can be retrieved.
@0x5bfa This will fix the issue, but is there any other better way?

Reply:
I'm not good at Task class.
Does this mean that the control was not loaded until a little time passed?

Reply:
> Does this mean that the control was not loaded until a little time passed?

Yes. I don't know why though.